pkgs/by-name/vi/vikunja/package.nix

```diff
@@ -121,8 +69,10 @@ buildGoModule {
 
   vendorHash = "sha256-d4AeQEAtPqMDe5a5aKhCe3i3pDXAMZJkJXxfcAFTx7A=";
 
+  inherit frontend;
+
   prePatch = ''
-    cp -r ${frontend} frontend/dist
+    cp -r $frontend frontend/dist
```

Comment:
Relying on bashrc here is not that nice. We should probably use finalAttrs if overrides are important.

Reply:
This is in a `buildGoModule` derivation without support for function based attrs :(